packages/widget-chat/src/selectors/notifications.js

```diff
@@ -0,0 +1,24 @@
+import {createSelector} from 'reselect';
```

Comment:
Using selectors to merge from different portions of the store. Not sure if this is the best way to do it, but it seemed better than trying to map them all in the component itself.